src/core_plugins/kibana/server/routes/api/ingest/register_fields.js

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash';
+import handleESError from '../../../lib/handle_es_error';
+
+export function registerFields(server) {
+  server.route({
+    path: '/api/kibana/{indices}/fields',
```

Comment:
Yeah, I gave it this name because I plan to expand it to provide all the field info Kibana needs once I [remove the mapping cache](https://github.com/elastic/kibana/pull/6648). That said, I agree it could be misleading atm so I'm open to giving it a different name in the interim. Any good ideas? `field_info` maybe? `field_stats` came to mind but seems like a bad idea since I'm filtering out most of the field_stats info.